build.go

```diff
@@ -304,7 +304,8 @@ func copyResourcesLinux(qmakePath string, target string) error {
 	}
 	// Everything read only except by owner
 	// Copy over some Qt DLLs
-	err := copyAndChmodEachToDirIfNotPresent(0644, filepath.Rel(filepath.Dir(qmakePath), "../lib"), target,
+	qmakeLibPath, _ := filepath.Rel(filepath.Dir(qmakePath), "../lib")
+	err := copyAndChmodEachToDirIfNotPresent(0644, qmakeLibPath, target,
```

Comment:
This was failing when there was an error, because filepath.Rel errors were not being captured.